docs/installation/local.md

```diff
@@ -95,7 +95,7 @@ python create_db.py
 # enter email and password
 python manage.py db stamp head
 ```
-
+**Note:** In case you are using Anaconda distribution for python, you may get an import error regarding `celery.signals` module. Please use the default python version while executing these steps in that case.
 
```

Comment:
Hey @abhishekvermasg , Can you also specify about how to activate the default python version and not use the anaconda version in the terminal ?
That would help the people who may not know how to activate the default version instead of Anaconda.